src/test/resources/features/3.7.x.feature

```diff
@@ -7,4 +7,13 @@ Feature: 3.7.x
     Examples: 
       | issueNumber | subtest | datasrc | args | expectation |
 
+#| 992 | 1 | "github992" | "-t {datasrc}/ff_del.xml" | "summary:totalErrors=1,summary:totalWarnings=5,summary:productValidation:failed=1,summary:messageTypes:error.table.field_value_data_type_mismatch=1,summary:messageTypes:warning.table.field_value_format_precision_mismatch=1,summary:messageTypes:warning.table.field_value_format_specifier_mismatch=4,summary:messageTypes:warning.table.field_value_too_long=1" |
+#| 1028 | 1 | "github1028" | "-t {datasrc}" |  |
+#| 1066 | 1 | "github1066" | "-t {datasrc}" |  |
```

Comment:
If you added these (I see test content for this ticket) and you changed your mind, then delete all of it. If you modified it and changed your mind, then it should be reverted to its original form or pick up fixes from a merge or rebase.